src/main/webapp/data/userMapData.json

```diff
@@ -684,17 +704,20 @@
       "Miriam College High School",
       "Misamis Oriental Institute Of Science and Technology",
       "New Era University",
+      "Oneself",
```

Comment:
May not be a real institution/organization. Tagging @damithc for confirmation.

Reply:
@chowyb @damithc  What about aGreatlife at line no. 690 ?